### ISSUE TITLE
Change run_wavelengths verbose default to False

### ISSUE THREAD

**Issue by andypbarrett** [robintw/Py6S#57]
I'm just getting started with Py6S version 1.7.2.  When I run the following: 

```
s = SixS()
wavelengths, results = SixSHelpers.Wavelengths.run_vnir(s, output_name="pixel_radiance")
# Plot these results, with the y axis label set to "Pixel Radiance"
SixSHelpers.Wavelengths.plot_wavelengths(wavelengths, results, "Pixel Radiance");
```
I get a lengthy output to stdout. 

It took some digging but I found run_wavelengths has verbose=True.  I wonder if this would be better with a default False or at least some documentation on how to suppress the output when running run_vnir etc.

I'm happy to attempt a modification and pull request.

**Comment by robintw** [comment]
Hi - sorry for the slow reply, the lockdown is making life rather chaotic at the moment.

Yes, that probably shouldn't be set to True as default. I'd be very happy to accept a PR if you're happy to submit one.

**Comment by robintw** [comment]
Fixed in 222f7df.
